e2e/specs/site.go

```diff
@@ -5,6 +5,13 @@ import (
 	"github.com/gravitational/robotest/e2e/model/ui"
 	sitemodel "github.com/gravitational/robotest/e2e/model/ui/site"
 
+	libdefaults "github.com/gravitational/robotest/lib/defaults"
```

Comment:
you can also drop the redundant alias - using this as `defaults.RetryAttempts` is fine as long as the package does not clash with another